python/lsst/cp/pipe/makeBrighterFatterKernel.py

```diff
@@ -159,9 +164,31 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
         inputs = butlerQC.get(inputRefs)
 
         # Use the dimensions to set calib/provenance information.
-        inputs['inputDims'] = dict(inputRefs.inputPtc.dataId.required)
+        # inputs['inputDims'] = dict(inputRefs.inputPtc.dataId.required)
+
+        # Add calibration provenance info to header.
+        if self.config.doHeaderProvenance:
+
+            for inputName in sorted(list(inputs.keys())):
+                kwargs = dict()
+                reference = getattr(inputRefs, inputName, None)
+
+                if reference is not None and hasattr(reference, "run"):
+                    runKey = f"LSST CALIB RUN {inputName.upper()}"
```

Comment:
Does this not have the same issue as the linearity, or is the issue that the string "linearity" is long enough to trigger the issue?